docs/features/configurations.md

```diff
@@ -0,0 +1,141 @@
+# Configurations
+Atmosphère provides a variety of customizable configurations to better adjust to users' needs.
+
+## BCT.ini
+This is the configuration file used by Fusée.
+This file is located under the `/atmosphere/config/` folder on your SD card and a default template can be found inside the `/atmosphere/config_templates/` folder.
+
+### Adding a Custom Boot Splashscreen
+Add the following lines to BCT.ini and change the value of `custom_splash` to the actual path and filename of your boot splashscreen:
+```
+[stage2]
+custom_splash = /path/to/your/bootlogo.bmp
+```
+
+The boot splashscreen must be a BMP file, it must be 720x1280 (1280x720 rotated 90 degrees left/counterclockwise/anti-clockwise) resolution, and be in 32-bit ARGB format. You can use image editing software such as GIMP or Photoshop to export the image in this format.
+
+### Configuring "nogc" Protection
+"nogc" is a feature provided by Fusée-secondary which disables the Nintendo Switch's Game Card reader. Its purpose is to prevent the reader from being updated when the console has been updated, without burning fuses, from a lower firmware version. More specifically, from firmware versions 4.0.0 or 9.0.0 which introduced updates to the Game Card reader's firmware. By default, Atmosphère will protect the Game Card reader automatically, but you are free to change it.
+
+To change its functionality, add the following line to the `stratosphere` section and change the value of `X` according to the following list:
+```
+nogc = X
+```
+```
+1 = force-enable nogc, so Atmosphère will always disable the Game Card reader.
+0 = force-disable nogc, so Atmosphère will always enable the Game Card reader.
+```
+
+### NCM opt-in
+Atmosphère provides a reimplementation of the [ncm](../components/modules/ncm.md) system module, but currently this is not enabled by default. If you wish to enable this reimplementation add the following line to the `stratosphere` section:
+```
+enable_ncm = 1
+```
+
+### Logging
+Add the following lines to BCT.ini and change the value of `X` according to the following list:
+```
+[config]
+log_level = X
+```
+```
+0 = NONE
+1 = ERROR
+2 = WARNING
+3 = MANDATORY
+4 = INFO
+5 = DEBUG
+```
+
```

Comment:
Either be complete and list all the config items, or add a mention that there are more that are listed in the config template.